conda-recipe/meta.yaml

```diff
@@ -29,20 +29,17 @@ requirements:
     - humanize
     - numpy
     - pytz
+    - boltons
 
 test:
   requires:
     - nslsii_dev_configuration
 
-  # Python imports
   imports:
+    # Import all the packages
     - metadatastore
-    - metadatastore.commands
```

Comment:
Further evidence of us being sloppy with the combination of conda-recipe + setup.py + library thrashing is that in the setup.py we are adding a package that does not exist... `metadatastore.utils`

https://github.com/NSLS-II/metadatastore/blob/master/setup.py#L22